.github/actions/run-mutations-testing/stryker-config.json

```diff
@@ -0,0 +1,23 @@
+{
+    "stryker-config": {
+        "concurrency": 2,
+        "mutation-level": "Standard",
+        "language-version": "latest",
+        "additional-timeout": 5000,
+        "mutate": ["**/*"],
```

Comment:
Is there anything its worth us excluding from the mutations for speed/because it will just fail? (Geniune question not saying you should) e.g. Program.cs

Reply:
That basically should already happen.

I.e. in `Program.cs`'s instance, there is 0% coverage for it, so no mutations are created because all mutations would just pass. Therefore it gets a score of 0% (because all mutations _would_ pass), but the "Survived" count is 0 and instead they'll be listed as "No Coverage".

This is the [fastest way](https://stryker-mutator.io/docs/stryker-net/configuration/#coverage-analysis-string) for it to run as well that I can see.